.idea/JMRI.iml

```diff
@@ -47,5 +47,6 @@
       </library>
     </orderEntry>
     <orderEntry type="library" name="lib" level="project" />
+    <orderEntry type="library" name="Python 2.7 interpreter library" level="application" />
```

Comment:
Is this change deliberate? I ask since sometimes these IDE file gets updated by misstake.